### PR TITLE
Menu: describe 'url' and 'linkAttr'

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/_hidden/dxMenuItem/linkAttr.md [hunk 1]
```diff
@@ -5,7 +5,123 @@ default: {}
...
+Specifies link attributes for the [url](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/items/#url) option.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+You can assign all attributes described in the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes" target="_blank">Attributes</a> article except for the `href` attribute. Use the [url](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/items/#url) property to specify a link instead.
+
+[note]
+
+- If you assign a `target: '_blank'` attribute, add a `rel: 'noopener'` attribute to avoid <a href="https://developer.chrome.com/en/docs/lighthouse/best-practices/external-anchors-use-rel-noopener/" target="_blank">security issues</a>.
```
Comment (Krijovnick) [inline api-reference/_hidden/dxMenuItem/linkAttr.md]: why we mention about 'noopener', but not about 'noreferrer' as in the article?

File: api-reference/_hidden/dxMenuItem/linkAttr.md [hunk 2]
```diff
@@ -5,7 +5,123 @@ default: {}
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies link attributes for the [url](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/items/#url) option.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+You can assign all attributes described in the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes" target="_blank">Attributes</a> article except for the `href` attribute. Use the [url](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/items/#url) property to specify a link instead.
```
Comment (Krijovnick) [inline api-reference/_hidden/dxMenuItem/linkAttr.md]: We don't recommend assign all attributes. Global attributes is not included in this list, except id/class and aria-*

**Review by Krijovnick — APPROVED** [review]

File: api-reference/_hidden/dxMenuItem/linkAttr.md [hunk 3]
```diff
@@ -5,7 +5,129 @@ default: {}
...
 ---
-<!-- Description goes here -->
\ No newline at end of file
+You can assign all attributes described in the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes" target="_blank">Attributes</a> article except for the following:
+
+- `href`    
+Use the [url](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/items/#url) property to specify a link instead.
+
+- <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes" target="_blank">Global attributes</a>    
+You can specify only `id`, `class` and <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes" target="_blank">aria-*</a> global attributes.
```
Comment (dirk-pieterse) [inline api-reference/_hidden/dxMenuItem/linkAttr.md]:
```suggestion
You can specify only `id`, `class`, and <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes" target="_blank">aria-*</a> global attributes.
```

File: api-reference/_hidden/dxMenuItem/url.md [hunk 4]
```diff
@@ -4,7 +4,229 @@ type: String
...
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a web address to be opened when a user clicks on an item.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+The following code opens a link in the same frame an item was clicked:
```
Comment (dirk-pieterse) [inline api-reference/_hidden/dxMenuItem/url.md]:
```suggestion
The following code opens a link in the same frame the item was clicked:
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**